config/src/config/network_config.rs

```diff
@@ -20,7 +20,9 @@ use std::{
 #[derive(Debug, Deserialize, PartialEq, Serialize)]
 #[serde(default)]
 pub struct NetworkConfig {
-    pub peer_id: String,
+    #[serde(serialize_with = "account_address::serialize_as_str")]
+    #[serde(deserialize_with = "account_address::deserialize_as_str")]
```

Comment:
We should take a look at trying to improve the ser/de impl on AccountAddress so that it can handle both these cases without requiring the {ser,de}_as_str functions. Not something you need to fix just make a note :)

Reply:
That would be stellar...

Reply:
this has been replaced by implementing the serialize / deserialize trait.